src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -524,7 +523,7 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
     (isFacStaff ? (
       <Typography align="left" className={styles.note}>
         NOTE: To update your data, please contact{' '}
-        <a style={{ color: gordonColors.primary.blue }} href="mailto: hr@gordon.edu">
+        <a className={styles.note_link} href="mailto: hr@gordon.edu">
```

Comment:
I replaced it with something like this: 
```className={`${'gc360_text_link'} ${styles.note_link}`}``` 
since I added to the gc350_text_link with bold and a transition effect, which I think makes these links look better in this context.  I then just removed the color overrides from styles.note_link, idk if this is better, but I do think the bold and transition looks better on the page.